LadybugTools_Engine/Python/src/ladybugtools_toolkit/wind.py

```diff
@@ -1675,7 +1675,7 @@ def plot_windmatrix(
         if other_data is None:
             other_data = self.ws
 
-        title = self.source
+        title = self.source if self.source is not None else "title"
```

Comment:
```suggestion
        title = self.source if self.source is not None else ""
```
It is better to initialise the title as an empty string (if the source is None).
This way you won't get titles looking like:
```
title
user_created_title
```